tests/test_backup.py

```diff
@@ -71,17 +73,66 @@ def test_rsync_dest_trailing_slash(temp_dir):
     dest1 = Path(temp_dir) / "dest1"
     dest2 = Path(temp_dir) / "dest2"
     # manager will create dest1 folder
-    manager = BackupManager(str(dest1), backup_utils.backup_logger)
+    manager = BackupManager(str(dest1))
     # dest_trailing_slash=True will create dest2
     manager.call_rsync(dest1, dest2, dest_trailing_slash=True)
     assert dest2.exists()
 
 
+def test_rsync_legacy_progress(monkeypatch, temp_dir, capfd):
```

Comment:
I think `temp_dir` is deprecated in favor of `tmp_path`
```suggestion
def test_rsync_legacy_progress(monkeypatch, tmp_path, capfd):
```

Reply:
ok, thanks, will adapt in all instances.